### PR TITLE
Fix work with naming argument in method add

### PR DESCRIPTION
**Pull request by ikrivosheev**
## What do these changes do?

`CIMultiDict.add` may be called with keyword arguments

## Are there changes in behavior for the user?

Restores the behaviour from 4.6.1

## Related issue number

fixes https://github.com/aio-libs/multidict/issues/421

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: multidict/_multidict.c [hunk 1]
```diff
@@ -639,12 +639,15 @@ multidict_tp_init(MultiDictObject *self, PyObject *args, PyObject *kwds)
...
 static PyObject *
-multidict_add(MultiDictObject *self, PyObject *args)
+multidict_add(MultiDictObject *self, PyObject *args, PyObject *kwds)
 {
     PyObject *key = NULL,
              *val = NULL;
 
-    if (!PyArg_UnpackTuple(args, "set", 2, 2, &key, &val)) {
+    static char *kwlist[] = {"key", "value", NULL};
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "OO:set",
```
Comment (asvetlov) [inline multidict/_multidict.c]:
```suggestion
    if (!PyArg_ParseTupleAndKeywords(args, kwds, "OO:add",
```

**Comment by asvetlov** [comment]
Thanks for the PR.
Could you please fix the function name, actually it is `add` instead of `set`?

File: multidict/_multidict.c (hunk 1 above)
Comment (ikrivosheev) [reply multidict/_multidict.c]: Yes, fix.

**Comment by ikrivosheev** [comment]
@asvetlov, done

**Pull request merged**